src/js/components/Navigation/ShowMoreFooter.jsx

```diff
@@ -9,20 +9,26 @@ import { renderLog } from '../../utils/logging';
 class ShowMoreFooter extends React.Component {
   static propTypes = {
     classes: PropTypes.object,
+    measureWeVoteId: PropTypes.string,
```

Comment:
Hi @uroshima, is measureWeVoteId necessary given you can pass in the actual "showMoreLink", and make that link anything you want? I think passing in measureWeVoteId is redundant.